trino/README.md

```diff
@@ -0,0 +1,59 @@
+# Agent Check: Trino
+
+## Overview
+
+This check monitors [trino][1].
```

Comment:
```suggestion
This check monitors [Trino][1].
```
While the official Trino logo is stylized with lower case, their webpage uses upper case for most instances of Trino.